mindocr/models/base_model.py

```diff
@@ -61,7 +61,7 @@ def construct(self, *args):
         nout = self.neck(bout)
 
         if len(args) > 1:
-            hout = self.head(nout, args[1:])
+            hout = self.head(nout, *args[1:])
```

Comment:
It conflicts the required input format (Tuple or List) in RobustScanner head https://github.com/mindspore-lab/mindocr/blob/main/mindocr/models/heads/rec_robustscanner_head.py#L695C33-L695C40

Reply:
We need to unify the form for parsing multi inputs in head. Since  all these head modules use `inputs` and `targets` as input args, I would suggest defining `targets` as type Tuple or List to allow multiple targets.

Reply:
Use tuple as input may introduce some bug (e.g. training memory keep growth) in Mindspore old version.  I suggest to use Tensor input only for the nn.Cell. and `def construct(self, inputs, targets)` can be changed to `def construct(self, inputs, *targets)` simply. But use list of Tensor as input might also be ok in current Mindspore version.

Reply:
Updated